features/fields.feature

```diff
@@ -0,0 +1,305 @@
+
+  Rule: Wanneer velden van polymorfe gegevensgroep wordt gevraagd, wordt altijd het 'type' veld van de gegevensgroep terug gegeven
+    - wanneer het gegeven geen waarde heeft (ook niet onbekend) wordt de gegevensgroep niet geleverd en dus ook 'type' niet teruggegeven
+
+    Abstract Scenario: Gevraagd veld is een polymorfe gegevensgroep van de persoon
+      Gegeven de persoon heeft de volgende 'geboorte' gegevens
+      | naam  | waarde      |
+      | datum | <GBA datum> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 999995078                          |
+      | fields              | burgerservicenummer,geboorte.datum |
+      Dan heeft de persoon met burgerservicenummer '999995078' alleen de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999995078 |
+      En heeft de persoon met burgerservicenummer '999995078' alleen de volgende 'geboorte' gegevens
+      | naam           | waarde     |
+      | datum.type     | <type>     |
+      | datum.datum    | <datum>    |
+      | datum.jaar     | <jaar>     |
+      | datum.maand    | <maand>    |
+      | datum.onbekend | <onbekend> |
+
+      Voorbeelden:
+      | GBA datum | type           | datum      | jaar | maand | onbekend |
+      | 20020220  | Datum          | 2002-02-20 |      |       |          |
+      | 20020200  | JaarMaandDatum |            | 2002 | 2     |          |
+      | 20020000  | JaarDatum      |            | 2002 |       |          |
+      | 00000000  | OnbekendDatum  |            |      |       | true     |
+
+    Abstract Scenario: Gevraagd veld is een gegeven van een andere type van de polymorfe gegevensgroep
+      Gegeven de persoon heeft de volgende 'geboorte' gegevens
+      | naam  | waarde      |
+      | datum | <GBA datum> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                   |
+      | type                | RaadpleegMetBurgerservicenummer          |
+      | burgerservicenummer | 999995078                                |
+      | fields              | burgerservicenummer,geboorte.datum.datum |
+      Dan heeft de persoon met burgerservicenummer '999995078' alleen de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999995078 |
+      En heeft de persoon met burgerservicenummer '999995078' alleen de volgende 'geboorte' gegevens
+      | naam       | waarde |
+      | datum.type | <type> |
+
+      Voorbeelden:
+      | GBA datum | type           |
+      | 20020200  | JaarMaandDatum |
+      | 20020000  | JaarDatum      |
+      | 00000000  | OnbekendDatum  |
+
+    Abstract Scenario: Gevraagd veld is een polymorfe gegevensgroep van de persoon en heeft geen waarde

```

Comment:
Waarom is hier een separaat scenario van gemaakt? Dit wijkt toch eigenlijk niet af van het scenario in regel 132.

Reply:
Dit illustreert dat de regel "wanneer het gegeven geen waarde heeft (ook niet onbekend) wordt de gegevensgroep niet geleverd en dus ook 'type' niet teruggegeven". Dit is dus een specifieke uitzondering op de regel "wordt altijd het 'type' veld van de gegevensgroep terug gegeven".

Het scenario op regel 132 toont het niet leveren van "normale" velden zonder discriminatoir